.github/workflows/publish-sdk.yml

```diff
@@ -0,0 +1,149 @@
+        working-directory: sdk
+        run: |
+            echo "STORE_PATH=$(pnpm store path)" >> $GITHUB_OUTPUT
+      - name: Setup pnpm cache
+        uses: actions/cache@v4
+        with:
+          path: ${{ steps.pnpm-cache.outputs.STORE_PATH }}
+          key: ${{ runner.os }}_${{ matrix.settings.target }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}_${{ matrix.settings.target }}-pnpm-store-
+      - name: Cache cargo
+        uses: actions/cache@v4
+        with:
+          path: |
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            .cargo-cache
+            target/
+          key: ${{ matrix.settings.target }}-cargo-${{ matrix.settings.host }}
+      - name: Install dependencies and buld
+        working-directory: sdk
+        run: pnpm install --frozen-lockfile
+      - name: Build
+        working-directory: sdk
+        run: ${{ matrix.settings.build }}
+        shell: bash
+      - name: Upload artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: bindings-${{ matrix.settings.target }}
+          path: ${{ env.FLIGHT_SQL_PATH }}${{ env.APP_NAME }}.*.node
+          if-no-files-found: error
+  publish:
+    name: Publish @ceramic-sdk packages
+    runs-on: ubuntu-latest
+    needs:
+      - build
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - name: Install pnpm
+        id: pnpm-install
+        uses: pnpm/action-setup@v3
+        with:
+          version: 9.8.0
+          run_install: false
+      - name: Get pnpm store directory
+        id: pnpm-cache
+        shell: bash
+        working-directory: sdk
+        run: |
+            echo "STORE_PATH=$(pnpm store path)" >> $GITHUB_OUTPUT
+      - name: Setup pnpm cache
+        uses: actions/cache@v4
+        with:
+          path: ${{ steps.pnpm-cache.outputs.STORE_PATH }}
+          key: ${{ runner.os }}_publish-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}_publish-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
+      - name: Install dependencies
+        working-directory: ${{ env.FLIGHT_SQL_PATH }}
+        run: pnpm install --frozen-lockfile
+      - name: Download all artifacts
+        uses: actions/download-artifact@v4
+        with:
+          path: ${{ env.FLIGHT_SQL_PATH }}/artifacts
+      - name: Move artifacts
+        working-directory: ${{ env.FLIGHT_SQL_PATH }}
+        run: pnpm artifacts
+      - name: List packages
+        working-directory: ${{ env.FLIGHT_SQL_PATH }}
+        run: ls -R ./npm
+        shell: bash
+      - name: Publish
+        working-directory: sdk
+        run: |
+          COMMIT_MESSAGE=$(git log -1 --pretty=%B)
+          if [[ $(echo "$COMMIT_MESSAGE" | grep 'chore(sdk): version v') ]]; then
```

Comment:
Should we move this step earlier on in the workflow so that other steps don't run if this isn't a release commit?